package.json

```diff
@@ -4,7 +4,7 @@
     "description": "",
     "main": "src/index.ts",
     "scripts": {
-      "build": "node esbuild.js && extism-js dist/index.js -i src/index.d.ts -o plugin.wasm"
+      "build": "node esbuild.js && extism-js dist/index.js -i src/index.d.ts -o dist/plugin.wasm"
```

Comment:
Let's also remove `typescript` from the devDependencies

Reply:
good call!